<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCortoPlazo</class>
 <widget class="QWidget" name="FormCortoPlazo">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evaluar Escenario</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Seleccione las matrices que desea utilizar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="4">
      <widget class="QListWidget" name="listAgregar">
       <item>
        <property name="text">
         <string>I</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>T</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>O</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="2" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #358ccb; color: #fff; font-weight: bold; height: 30px; border: none; border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Seleccionar</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Matrices:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="4">
      <widget class="QListWidget" name="listSeleccionado"/>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: bold; height: 30px; border: none; border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-weight: bold; height: 30px; border: none; border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font-weight:bold;</string>
       </property>
       <property name="text">
        <string>Matrices Seleccionadas:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #358ccb; color: #fff; font-weight: bold; height: 30px; border: none; border-radius: 5px;</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
